City-of-Helsinki/drupal-helfi-etusivu
# UHF-7492: sitemap

# [UHF-7492](https://helsinkisolutionoffice.atlassian.net/browse/UHF-7492)

## What was done
Show sitemap.xml on /fi/sitemap.xml instead of /sitemap.xml by overriding SitemapPathProcessor

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-7492_sitemap`
  * `make fresh`
* Run `make drush-cr`

## How to test
Go to `https://helfi-etusivu.docker.so/fi/sitemap.xml``
- You should see the sitemap



[UHF-7492]: https://helsinkisolutionoffice.atlassian.net/browse/UHF-7492?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
